libportability-gfx/src/impls.rs

```diff
@@ -4384,6 +4385,14 @@ pub extern "C" fn gfxGetPhysicalDeviceSurfacePresentModesKHR(
     code
 }
 
+#[inline]
+pub extern "C" fn gfxGetPhysicalDeviceWin32PresentationSupportKHR(
+    adapter: VkPhysicalDevice,
```

Comment:
note that we'd want to indicate that the arguments are unused by prepending their names with `_`